### ISSUE TITLE
Tibiotalar (ankle) arthritis

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11204]
**Preferred term label:**
Tibiotalar (ankle) arthritis

**Synonyms**

**Definition (free text, please give PubMed ID)**
AP, oblique, or lateral view x-ray of the ankle showing radiographic evidence of osteoarthritis such as joint space narrowing, subchondral sclerosis, subchondral cysts, and/or osteophyte formation in the ankle joint

https://uts.nlm.nih.gov/uts/umls/concept/C0238650

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Osteochondral lesion of the talus (OLT)

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Duplicate of https://github.com/obophenotype/human-phenotype-ontology/issues/11214
